src/app/cli/src/client.ml

```diff
@@ -429,30 +429,40 @@ let user_command (body_args : User_command_payload.Body.t Command.Param.t)
            (Currency.Fee.to_int Cli_lib.Fee.default_transaction))
       (optional txn_fee)
   in
-  let flag = Args.zip3 body_args Cli_lib.Flag.privkey_read_path amount_flag in
+  let nonce_flag =
+    flag "nonce"
+      ~doc:
+        "NONCE Nonce that you would like to set for your transaction \
+         (default: nonce of your account on the best ledger)"
+      (optional txn_nonce)
+  in
+  let flag =
+    Args.zip4 body_args Cli_lib.Flag.privkey_read_path amount_flag nonce_flag
+  in
   Command.async ~summary
     (Cli_lib.Background_daemon.init flag
-       ~f:(fun port (body, from_account, fee) ->
+       ~f:(fun port (body, _from_account, fee_opt, nonce_opt) ->
          let open Deferred.Let_syntax in
-         let%bind sender_kp =
-           Secrets.Keypair.Terminal_stdin.read_exn from_account
-         in
-         let%bind nonce = get_nonce_exn sender_kp.public_key port in
-         let fee = Option.value ~default:(Currency.Fee.of_int 1) fee in
-         let payload : User_command.Payload.t =
-           User_command.Payload.create ~fee ~nonce
-             ~memo:User_command_memo.dummy ~body
+         let sender_kp = Genesis_ledger.largest_account_keypair_exn () in
```

Comment:
Did you do some debugging and forget to revert it? Obviously we have to send from the requested account.